dts/riscv/riscv32-fe310.dtsi

```diff
@@ -93,6 +93,15 @@
 			label = "gpio_0";
 			status = "disabled";
 			#gpio-cells = <2>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges;
+
+			pinctrl: pinctrl@10012000 {
```

Comment:
This doesn't look right; I understood the unit address had to match the first cell of the `reg` property.

Reply:
Good catch, missed to update that in my copy/paste.